Managers/MovableMan.cpp

```diff
@@ -1356,25 +1356,26 @@ bool MovableMan::RemoveMO(MovableObject *pMOToRem)
 
 
 //////////////////////////////////////////////////////////////////////////////////////////
-// Method:          KillAllActors
+// Method:          KillAllEnemyActors
 //////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Kills and destroys all actors of a specific team.
+// Description:     Kills and destroys all enemies of a specific team.
 
-int MovableMan::KillAllActors(int exceptTeam)
+int MovableMan::KillAllEnemyActors(int exceptTeam)
 {
     int killCount = 0;
-    AHuman *pHuman = 0;
 
     // Kill all regular Actors
     for (deque<Actor *>::iterator aIt = m_Actors.begin(); aIt != m_Actors.end(); ++aIt)
     {
         if ((*aIt)->GetTeam() != exceptTeam)
         {
-            // Blow up the head of humanoids, for effect
-            if (pHuman = dynamic_cast<AHuman *>(*aIt))
-                pHuman->GetHead()->GibThis();
-            else
-                (*aIt)->GibThis();
+            // Blow up the heads of humanoids, for effect.
```

Comment:
I guess you forgot to tab out the comment
Also, since you're poking at it already, you should change `pHuman` to just `human` or something. We don't use `p` in our naming.

All this applies below too.